client/src/components/ui/tag/TagGroup.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { Tag, Input, Icon, Row } from 'antd';
+import { TweenOneGroup } from 'rc-tween-one';
+
+class TagGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tags: this.props.tags,
+      inputVisible: false,
+      inputValue: '',
+    };
+  }
+
+  componentDidUpdate = () => {
+    const { tags } = this.props;
+    if (tags != this.state.tags) {
```

Comment:
How about using '!==' instead of '!='? It will not make an error but just for stability.
Also you used !== in line 24, so I think it is better to choose one of '!==' or '!='.